packages/ui-components/src/components/text-field/text-field.base.scss

```diff
@@ -170,6 +176,34 @@ input {
 			--icon-color: var(--text-color-input-focused);
 		}
 	}
+
+	.right-slot-container {
+		position: absolute;
+		top: 0;
+		right: 0;
+		height: 100%;
+		width: $right-slot-width;
+		overflow: hidden;
+		
+		::slotted(kv-svg-icon) {
+			--icon-width: 16px;
```

Comment:
we need to override the default configuration with the same values?

Reply:
The css added is necessary in order to place the icon. After checking the possibility of removing the repeated properties, the icon would move out of place.